mslib/mscolab/server.py

```diff
@@ -351,6 +352,40 @@ def get_user():
     return json.dumps({'user': {'id': g.user.id, 'username': g.user.username}})
 
 
+@APP.route('/upload_profile_image', methods=["POST"])
+@verify_user
+def upload_profile_image():
+    user_id = request.form['user_id']
+    file = request.files['image']
+    if not file:
+        return jsonify({'message': 'No file provided or invalid file type'}), 400
+    if not file.mimetype.startswith('image/'):
+        return jsonify({'message': 'Invalid file type'}), 400
+    if file.content_length > mscolab_settings.MAX_UPLOAD_SIZE:
+        return jsonify({'message': 'File too large'}), 413
+
+    file_token = secrets.token_urlsafe(20)
+    success, message = fm.save_user_profile_image(user_id, file, file_token, APP.config['UPLOAD_FOLDER'])
+    if success:
+        return jsonify({'message': message}), 200
+    else:
+        return jsonify({'message': message}), 400
+
+
+@APP.route('/fetch_profile_image', methods=["GET"])
+@verify_user
+def fetch_profile_image():
+    user_id = request.form['user_id']
+    user = User.query.get(user_id)
+    if user and user.profile_image_path:
+        base_directory = mscolab_settings.UPLOAD_FOLDER
+        base_path = os.path.join(base_directory, 'profile')
```

Comment:
This hard-codes the profile path instead of using the config value. This is also the case in other places.